db/postgres/schema/core.sql

```diff
@@ -0,0 +1,50 @@
+CREATE TABLE banks (
+  id INT PRIMARY KEY,
+  bank_name VARCHAR(50) NOT NULL
+);
+
+CREATE TABLE deadlines (
+  id INT PRIMARY KEY,
+  payment_date VARCHAR(255) NOT NULL
+);
+
+CREATE TABLE payments(
+  id INT PRIMARY KEY,
+  amount INT NOT NULL,
+  payment_limit INT NOT NULL
+);
+
+CREATE TABLE companies (
+  id SERIAL PRIMARY KEY,
+  company_name VARCHAR(255) NOT NULL,
+  phone_num INT NOT NULL,
+  create_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE TABLE users (
+  id SERIAL PRIMARY KEY,
+  username VARCHAR(255) NOT NULL,
+  email VARCHAR(255) NOT NULL UNIQUE,
+  password_hash VARCHAR(255) NOT NULL,
+  deleted_at TIMESTAMPTZ NULL
+);
+
+CREATE TABLE bills (
+  id SERIAL PRIMARY KEY,
+  user_id INT NOT NULL REFERENCES users (id),
+  company_id INT NOT NULL REFERENCES companies (id),
+  bank_id INT NOT NULL REFERENCES banks (id),
+  deadline_id INT NOT NULL REFERENCES deadlines (id),
+  payment_id INT NOT NULL REFERENCES payments (id),
+  is_paid BOOLEAN DEFAULT FALSE NOT NULL,
+  paid_at TIMESTAMPTZ,
```

Comment:
```suggestion
  paid_at TIMESTAMPTZ,
```
paid_atだけで目的は達せられそうに思った.